test/Fail/Issue4748c.err

```diff
@@ -1,3 +1,3 @@
 Issue4748c.agda:14,8-9
-(@0 x : A) (y : B x) → R is not usable at the required modality @ω
+(@0 x : A) (y : B x) → R is not usable at the required modality @++
```

Comment:
I think it should still print `@omega` here (and in similar cases) and not `@++`, because the testcase is about `@0`.
Also, why `@++` is not really clear to me here.

Reply:
But this means that the error message is silly, right? Any name/variable that's used anywhere should be usable with the identity modality (since if it occurs under another modality, that other modality is brought to the left as a left division, leaving again only the identity modality). So what the error message should tell you also/instead, is by what modality the name/variable *is* available.

Reply:
The current error draws attention to the wrong coordinate of the modality vector.  It is off in the _erasure_ dimension, not in the _variance_ dimension, so the error should state that it "is erased but should be non-erased".  @anyts suggestion would express this in technical terms, maybe there could be a loud-reading for the user stating the dimensions that are wrong...